docker/build-trigger.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Triggers an auto-build on dockerhub for the given source control reference.
+#
+# Example usage: ./build-trigger Tag 1.0.0 https://registry.hub.docker.com/u/scitran/reaper/trigger/11111111-2222-3333-4444-abcdefabcdef/
+function Main() {(
+  set -e
+
+  if [ $# -ne 3 ] ; then
+    >&2 echo "Usage: $( basename $0 ) <source-control-ref-type> <source-control-ref-value> <trigger-url>"
+    exit 1
+  fi
+
+  export SOURCE_CONTROL_REF_TYPE="${1}"
+  export SOURCE_CONTROL_REF_NAME="${2}"
+  export TRIGGER_URL="${3}"
+
+
+  if [ -z "${SOURCE_CONTROL_REF_TYPE}" ] ; then
+    >&2 echo "INFO: Source control reference type provided, skipping build trigger."
```

Comment:
Shouldn't this say "Source control reference type NOT provided"? If so, then this is also wrong in scitran/reaper.